pilot/pkg/proxy/envoy/v1/mock/discovery.go

```diff
@@ -285,6 +285,11 @@ func (sd *ServiceDiscovery) GetProxyServiceInstances(node model.Proxy) ([]*model
 	return out, sd.GetProxyServiceInstancesError
 }
 
+// GetProxyLabels ...
```

Comment:
If you must add GetProxyLabel instead of using the existing method - please implement it.

Reply:
Not implementable.